### ISSUE TITLE
Custom soundfonts

### ISSUE THREAD

**Issue by Bewelge** [Bewelge/MIDIano#37]
Explore if feasible. Might be too large / resource intensive

**Comment by spessasus** [comment]
Hi,

I might have [a solution for you :)](https://github.com/spessasus/SpessaSynth?tab=readme-ov-file#spessasynth_lib)

**Comment by Bewelge** [comment]
> Hi,
> 
> I might have [a solution for you :)](https://github.com/spessasus/SpessaSynth?tab=readme-ov-file#spessasynth_lib)

Wow, that does look interesting! Did you build this yourself from scratch? Will have to check it out! :)

**Comment by spessasus** [comment]
Yes, I've created this myself. Though MIDIano was an inspiration when creating the frontend for it ;-)
It would also resolve #60 #54 #55
And a higher quality soundfont could be used as it wouldn't use online samples for every key, but a proper .sf2 file.
